<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>205</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>327</width>
    <height>205</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>327</width>
    <height>205</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Conversor de Temperturas</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/icon.jpeg</normaloff>../img/icon.jpeg</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>317</width>
     <height>189</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Ingrese la temperatura y seleccione una escala</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="calor">
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo">
        <item>
         <property name="text">
          <string>Grados Celsius</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Grados Kelvin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Grados Fahrenheit</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="grados_celsius">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="grados_kelvin">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="grados_fahrenheit">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="boton_salir">
      <property name="text">
       <string>Salir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
